### PR TITLE
Week02: Fav Scientist & Cat Walk (unfinished)

### PR DESCRIPTION
**Pull request by peachy555**
So sorry about the late pull request. This submitted codes should all be able to run and complete minimum requirements for the homework. I will submit again during weekend, fully "completed".

Day07: Fav Scientist
Difficulty: 5
Just not used to using framework, but after sometimes fiddling around with the code, I feel a lot more confident using the framework.

Time taken: 5hrs

Day08: Cat Walk
Difficulty: 3.5
Taking some time to research about jQuery methods/functions.

Time taken: 3hrs

### CODE REVIEW

**Comment by peachy555** [comment]
Day 10: GA Bank
Difficulty: 2
A bit confused on how to deal with data received from <input>, but ok after that.

Time taken: 3.5 hrs

**Comment by peachy555** [comment]
Day10: GA Bank
Difficulty: 2
Not too difficult. A bit confusing at the start on how to handle data from <input> tag, but nothing much after that.

Time taken: 3 hrs

**Comment by peachy555** [comment]
Updated version of GA Bank.
Generalised all deposit & withdraw function using $(this) as handler.

**Comment by peachy555** [comment]
Day10 HW with overdraft protection

File: peach_tuntiwong/week02/day10/js/atm.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  var checkingBalanceUpdate = function(amount) {
+    checkingBalance += amount;
+    $("#checking-balance").html('$'+checkingBalance);
+    isChecking0Balance();
+  }
+
+  var savingsBalanceUpdate = function(amount) {
+    savingsBalance += amount;
+    $("#savings-balance").html('$'+savingsBalance);
```
Comment (textchimp) [inline peach_tuntiwong/week02/day10/js/atm.js]: Ideally, for Separation of Concerns, we'd like to move these jQuery lines that change the DOM out of the functions for banking logic.

**Comment by textchimp** [comment]
Nice work on the bank and the catWalk.

**Pull request merged**